src/domain/individualApplication/utils.ts

```diff
@@ -121,3 +124,47 @@ export const getFilteredCustomersData = (
     ];
   }, []);
 };
+
+interface LeaseDetails {
+  id: string;
+  berthComment: string;
+  berthDepth: number | null;
+  berthIsAccessible: boolean;
+  berthLength: number | null;
+  berthMooringType: BerthMooringType | null;
+  berthNum: string;
+  berthWidth: number | null;
+  electricity: boolean;
+  gate: boolean;
+  harborName: string;
+  lighting: boolean;
+  pierIdentifier: string;
+  wasteCollection: boolean;
+  water: boolean;
+}
+
+export const getOfferDetailsData = (
+  lease: BERTH_LEASE
+): Omit<OfferCardProps, 'handleDeleteLease'> => {
+  const leaseDetails: LeaseDetails | null = {
+    id: lease.id,
+    berthComment: lease.berth?.comment || '',
+    berthDepth: lease.berth?.berthType.depth || null,
+    berthIsAccessible: lease.berth?.isAccessible || false,
+    berthLength: lease.berth?.berthType.length || null,
+    berthMooringType: lease.berth?.berthType.mooringType || null,
+    berthNum: lease.berth?.number || '',
+    berthWidth: lease.berth?.berthType.width || null,
+    electricity: lease.berth?.pier.properties?.electricity || false,
+    gate: lease.berth?.pier.properties?.gate || false,
+    harborName: lease.berth?.pier.properties?.harbor.properties?.name || '',
+    lighting: lease.berth?.pier.properties?.lighting || false,
+    pierIdentifier: lease.berth?.pier.properties?.identifier || '',
+    wasteCollection: lease.berth?.pier.properties?.wasteCollection || false,
+    water: lease.berth?.pier.properties?.water || false,
+  };
+
+  return {
+    leaseDetails,
+  };
```

Comment:
Why don't we return  `leaseDetails` directly?
```suggestion
  return leaseDetails;
```

Reply:
My intention was that there would be a separate `billingDetails` object once we get that in order

Reply:
Gotcha!